tests/test_xjse_calendar.py

```diff
@@ -0,0 +1,61 @@
+from unittest import TestCase
+import pandas as pd
+from pytz import UTC
+
+from .test_trading_calendar import ExchangeCalendarTestBase
+from trading_calendars.trading_calendar import WEEKENDS
+from trading_calendars.exchange_calendar_xjse import XJSEExchangeCalendar
+
+
+class XJSECalendarTestCase(ExchangeCalendarTestBase, TestCase):
+
+    answer_key_filename = 'xjse'
+    calendar_class = XJSEExchangeCalendar
+
+    # The JSE is open from 09:00 to 17:00.
+    MAX_SESSION_HOURS = 8
+
+    HAVE_EARLY_CLOSES = False
+
+    # Johannesburg does not use daylight savings time: South Africa
+    # Standard Time (SAST) is observed all year.
```

Comment:
Looking at `ExchangeCalendarTestBase`, it looks like DAYLIGHT_SAVINGS_DATES defaults to `["2004-04-05", "2004-11-01"]`, so I don't think this is what you're looking for.  From quickly poking around other calendars that don't have daylight savings, I found that the Hong Kong calendar test case does this:
```
# Asia/Hong_Kong does not have daylight savings
    DAYLIGHT_SAVINGS_DATES = []
```

However, I'm not really sure what this test actually does because we don't define daylight savings anywhere else (maybe it's calculated from the given timezone, but then I'm not sure why this test passed when the default daylight savings dates were used here?)

Reply:
The test that uses these dates just checks the calculated open time for the next day's session: since this calendar's timezone doesn't apply daylight savings, the two open times it checks here are the same as all the others, so these checks are redundant.

Specifying this value as an empty list will essentially make the test a no-op. Neither of those possibilities seems super helpful, but maybe the latter will make it clearer that the test isn't doing anything useful.